src/server/server.js

```diff
@@ -51,6 +52,9 @@ async function boot(app) {
   if (!appConfig.isDev) {
     app.enable('trust proxy')
     app.use(expressEnforcesSsl())
+  }
+
+  if (process.env.REDIS_URL) {
```

Comment:
This kind of a change is a bit dangerous for production environments. Before, you could be sure that in production, if the REDIS_URL wasn't defined, the whole thing would crash and you would notice it. Now however, the app would silently use the in-memory session, which is not suitable for production.

I assume you did this change so that you could use the redis session store at development time? We could add a `|| !appConfig.isDev` here so it still forces you to Redis in prod, but allows you to use it in dev too, or we could just revert these lines

Reply:
Yeah, the point was to allow me to use Redis locally. I changed it to always use redis on prod.